test/help/sexpr-wasm.txt

```diff
@@ -1,25 +1,25 @@
 ;;; EXE: %(sexpr-wasm)s
```

Comment:
I will renamed those and the tests failed . with the output:
**** FAILED ****************************************************************** 
- help/sexpr-wasm.txt
    
- help/wasm-wast.txt

 I can't quite tell why